src/tester.rs

```diff
@@ -162,7 +162,7 @@ impl QuickCheck {
 
         let n_tests_passed = match self.quicktest(f) {
             Ok(n_tests_passed) => n_tests_passed,
-            Err(result) => panic!(result.failed_msg()),
+            Err(result) => panic!("{}", result.failed_msg()),
```

Comment:
Hmmm I don't think this is right... What's the motivation for this change?

Reply:
Like with `write!`, `print!` or good old `printf`, passing dynamic string-data as format string (`panic!` can format messages for some time now) is frowned upon. Current versions of rust will report a warning during compilation for this line, and probably an error in the future.

Reply:
Ah I see. Gotya. Yeah this is good, thanks for catching it!

Reply:
I'd be ok with this. But I doubt that we'll see anything merged in this repo in the near future.